client/src/pages/LyricsView/TrackPlaying.tsx

```diff
@@ -6,9 +6,10 @@ import { ITrackLyrics } from "../../types/trackLyrics";
 
 interface IProps {
   lyricDetails?: ITrackLyrics;
+  currentlyPlayingSong?: any;
```

Comment:
I think this can be `PlayingStatePlaying` (will need to be exported from `client\src\types\currentlyPlayingState.ts`)